coder-sdk/client.go

```diff
@@ -36,5 +32,7 @@ func (c *Client) http() (*http.Client, error) {
 			Secure:   c.BaseURL.Scheme == "https",
 		},
 	})
+	// TODO: Consider setting a timeout.
+	// TODO: Consider having a persistent client to allow for single connection.
```

Comment:
add bug reports for this? I think timeouts are great. 

I'd also be inclined in doing something to pool/reuse clients (e.g. https://github.com/deciduosity/utility/blob/main/http.go)

Reply:
Will do.

Using a 3rd party pooling here might be overkill, but it would be interesting in our services

Reply:
The pool seems to only be needed when you're playing with multiple transports, I think it's fine as is for now.